features/step_definitions/record_of_guidance_contents_steps.rb

```diff
@@ -135,16 +130,12 @@
 end
 
 Then(/^the record of guidance should include the details of the appointment$/) do
-  output_document = OutputDocument.new(@appointment_summary)
-
-  expected = {
-    appointment_date: output_document.appointment_date,
-    guider_first_name: output_document.guider_first_name,
-    guider_organisation: output_document.guider_organisation
-  }
+  expected = fixture(:populated_csv)
```

Comment:
we should assert only the fields we care about, not the entire csv row, otherwise unrelated changes will fail this assertion